Assets/Mirror/Discovery/NetworkDiscovery.cs

```diff
@@ -1,31 +1,30 @@
-﻿using UnityEngine;
+using UnityEngine;
 using System.Net;
-using System.Net.Sockets;
 using System;
-using System.Threading.Tasks;
 using UnityEngine.Events;
 
 namespace Mirror.Discovery
 {
     // Based on https://github.com/EnlightenedOne/MirrorNetworkDiscovery
     // forked from https://github.com/in0finite/MirrorNetworkDiscovery
     // Both are MIT Licensed
-    [System.Serializable]
+    [Serializable]
     public class ServerFoundUnityEvent : UnityEvent<ServerResponse> { };
 
     [DisallowMultipleComponent]
     [AddComponentMenu("Network/NetworkDiscovery")]
-    [HelpURL("https://mirror-networking.com/docs/Components/NetworkDiscovery.html")]
     public class NetworkDiscovery : NetworkDiscoveryBase<ServerRequest, ServerResponse>
     {
         #region Server
-        public long ServerId { get; private set; }
 
-        public ServerFoundUnityEvent ServerFound;
+        public long ServerId { get; private set; }
 
-        [Tooltip("Transport exposed for discovery")]
+        [Tooltip("Transport used for discovery")]
```

Comment:
This is a bit confusing,   we always use UDP broadcast for discovery,  not the transport.  We just make that transport discoverable, we don't actually use it.